src/script/E2EIdentity/E2EIdentityEnrollment.ts

```diff
@@ -262,7 +274,11 @@ export class E2EIHandler extends TypedEventEmitter<Events> {
     return oidcService.getUser();
   }
 
-  public async enroll(snoozable: boolean = true) {
+  public async enroll({snoozable = true, resetTimers = false}: {snoozable?: boolean; resetTimers?: boolean} = {}) {
+    if (resetTimers) {
+      this.enrollmentStore.clear.timer();
+    }
```

Comment:
This is for cases where we update the cert manually (from the UI), we reset the timer, so it won't fire sooner then needed (i.e. new cert will be valid for 90 days, no need to show the stored timer in 15mins)